### ISSUE TITLE
server actions: missing disable rules actions

### ISSUE THREAD

**Issue by florian-sanders** [florian-sanders/zed-stylelint#1]
Seems like `disable x rule next line` / `disable x rule entire file` are not available even though the server does support them.

Might be something to pass to the server during the initialization?

**Comment by florian-sanders** [comment]
I've been inspecting the VSCode Stylelint extension source code for a few days and I don't think I can fix the issue without modifying the server source code:

- if the editor sends requests with `fixall` or `source` code actions, then the server responds only with the `fixall` action,
- code actions for disable rules seem to only be available if the editor sends a request with no code action.

Zed always sends all code actions with its requests apparently so we never receives disable rules code actions.

I still need to confirm all of this but this is what I've gathered from logging within this part of the code:
https://github.com/stylelint/vscode-stylelint/blob/f2d2ac30237cb79b148aaa4076120d79475d47cc/src/server/modules/code-action.ts#L190

TLDR; the server code would some decoupling from VSCode behavior

**Comment by florian-sanders** [comment]
Fixed with #8
